test/e2e/scheduling/priorities.go

```diff
@@ -206,7 +206,7 @@ var _ = SIGDescribe("SchedulerPriorities [Serial]", func() {
 		}
 	})
 
-	It("Pod should perfer to scheduled to nodes pod can tolerate", func() {
+	It("Pod should prefer to scheduled to nodes pod can tolerate", func() {
```

Comment:
Have you tried to search perfer globally in the repo? Normally typo is caused by silly copy-paste.

Reply:
@resouer Found one more, and fixed.